ui/page/account/account_page.go

```diff
@@ -0,0 +1,272 @@
+		return
+	}
+
+	for _, acct := range accounts.Accounts {
+		if acct.Number == dcr.ImportedAccountNumber {
+			continue
+		}
+		walletAccounts = append(walletAccounts, acct)
+	}
+
+	pg.accounts = walletAccounts
+}
+
+// OnNavigatedTo is called when the page is about to be displayed and
+// may be used to initialize page features that are only relevant when
+// the page is displayed.
+// Part of the load.Page interface.
+func (pg *Page) OnNavigatedTo() {
+	pg.usdExchangeSet = false
+	if components.IsFetchExchangeRateAPIAllowed(pg.WL) {
+		pg.usdExchangeSet = pg.WL.AssetsManager.RateSource.Ready()
+		go func() {
+			rate, err := pg.Load.WL.FetchExchangeRate()
+			if err != nil {
+				log.Error(err)
+				return
+			}
+			pg.exchangeRate = rate
+			pg.ParentWindow().Reload()
+		}()
+	}
+}
+
+// Layout draws the page UI components into the provided layout context
+// to be eventually drawn on screen.
+// Part of the load.Page interface.
+func (pg *Page) Layout(gtx C) D {
+	return pg.Theme.List(pg.container).Layout(gtx, 1, func(gtx C, i int) D {
+		return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+			return components.UniformHorizontalInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(pg.headerLayout),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
+							return pg.accountsList.Layout(gtx, len(pg.accounts), func(gtx C, i int) D {
+								return pg.accountItem(gtx, pg.accounts[i], i == len(pg.accounts)-1)
+							})
+						})
+					}),
+				)
+			})
+		})
+	})
+}
+
+func (pg *Page) headerLayout(gtx C) D {
+	return layout.Inset{
+		Top:    values.MarginPadding24,
+		Bottom: values.MarginPadding12,
+	}.Layout(gtx, func(gtx C) D {
+		return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				txt := pg.Theme.Label(values.TextSize20, values.String(values.StrAccounts))
+				txt.Font.Weight = font.SemiBold
+				return txt.Layout(gtx)
+			}),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.E.Layout(gtx, pg.addAccountLayout)
+			}),
+		)
+	})
+}
+
+func (pg *Page) addAccountLayout(gtx C) D {
+	return cryptomaterial.LinearLayout{
+		Width:      cryptomaterial.WrapContent,
+		Height:     cryptomaterial.WrapContent,
+		Background: pg.Theme.Color.DefaultThemeColors().SurfaceHighlight,
+		Clickable:  pg.addAccountBtn,
+		Alignment:  layout.Middle,
+	}.Layout(gtx,
+		layout.Rigid(pg.Theme.Icons.AddIcon.Layout16dp),
+		layout.Rigid(func(gtx C) D {
+			txt := pg.Theme.Label(values.TextSize16, values.String(values.StrAddNewAccount))
+			txt.Color = pg.Theme.Color.DefaultThemeColors().Primary
+			txt.Font.Weight = font.SemiBold
+			return layout.Inset{
+				Left: values.MarginPadding8,
+			}.Layout(gtx, txt.Layout)
+		}),
+	)
+}
+
+func (pg *Page) accountItem(gtx C, account *sharedW.Account, isHiddenLine bool) D {
```

Comment:
- rename `accountItem` to `accountItemLayout`.
- rename `isHiddenLine` to `isLastItem`.

Also, here's one way I imagine the account item layout can match what's on figma:
```
widgets := []layout.FlexChild{
	layout.Rigid - spacer{height: 24px}
	layout.Rigid - layoutAmountWithLabel(accountName, accountBalance, 18px)
	layout.Rigid - spacer{height: 16px}
	layout.Rigid - layoutAmountWithLabel("Spendable", spendableBalance, 16px)
	layout.Rigid - spacer{height: 16px}
}

if !isLastItem { // add divider if not last item
	divider := inset{top: 12, bottom: 12}.layout(line)
	widgets = append(widgets, layout.Rigid - divider)
}

return layout.Flex{Axis: layout.Vertical}.Layout(gtx, widgets...)
```

The `layoutAmountWithLabel` fn referenced above will look something like:
```
layoutAmountWithLabel(labelText string, amount sharedW.AssetAmount, fontSize unit.Sp) {
	flex(horizontal, space between).layout(gtx, labelText, amountText, amountUsdText)
}
```

Both `labelText` and `amountText` will use the provided fontSize, while the amountUsdText will use the provided fontSize-2.

Reply:
The suggested code above has a flaw, will amend and update.